deploy/charts/operator-crds/templates/mcpexternalauthconfigs-crd.yaml

```diff
@@ -0,0 +1,5 @@
+{{- if .Values.crds.install.server }}
```

Comment:
IIRC, external auth can also be used with virtual MCPs, so this should be in OR with `.Values.crds.install.virtualMCP`